### ISSUE TITLE
Missing "cluster_log_conf" configuration in the "New-DatabricksCluster" cmdlet.

### ISSUE THREAD

**Issue by RomainLasso** [DataThirstLtd/azure.databricks.cicd.tools#65]
Hi @simondmorias 

The "**cluster_log_config**" parameter is not avalaible in the cmlet "New-DatabricksCluster" : https://docs.databricks.com/api/latest/clusters.html#clusterlogconf.

Configuration sample : 

``` json
{
    "cluster_name": "xxx",
    "spark_version": "5.3.x-scala2.11",
    "cluster_log_conf": {
        "dbfs": {
            "destination": "dbfs:/cluster-logs"
        }
    },     
    "cluster_id": "xxx"
}
```

It would be nice to add this parameter.

Romain

**Comment by AlexMasson** [comment]
Hi, is there any news about this issue ?
Thanks
Alexandre

**Comment by simondmorias** [comment]
Sorry been away for a couple of weeks. I will look Thursday or at the weekend latest.

________________________________

**Comment by simondmorias** [comment]
A new version have been released (1.1.21) which includes this for new clusters and on job clusters. I have also included the new instance pool id flag.
